test/lib/generation/commands/add_constant_command_test.dart

```diff
@@ -19,8 +19,8 @@ void main() {
 
     setUp(() {
       strategy = MockFSStrategy();
-      const1 = AddConstantCommand(const1V[0], const1V[1], const1V[2]);
-      const2 = AddConstantCommand(const2V[0], const2V[1], const2V[2]);
+      const1 = AddConstantCommand('UUID', const1V[0], const1V[1], const1V[2]);
```

Comment:
For this and other tests that contain the string `UUID` as a UUID, will that affect the test itself? 🤔

Reply:
Not for this unit test since I am not retrieving any imports or doing anything with the UUID, do you think I should update it?🤔